lib/workers/repository/process/lookup/index.ts

```diff
@@ -143,6 +146,24 @@ function getBucket(
   return 'non-major';
 }
 
+function hasPendingChecks(
+  config: LookupUpdateConfig,
+  release: Release
+): boolean {
+  if (!config.stabilityDays) {
+    return false;
+  }
+  if (!release?.releaseTimestamp) {
+    return false;
+  }
+  const releaseTimestamp = DateTime.fromISO(release.releaseTimestamp);
```

Comment:
Needs a check for `isValid`. How should we handle invalid date strings? treat them as undefined?
maybe log a debug message and ignore?

https://moment.github.io/luxon/docs/class/src/datetime.js~DateTime.html#instance-get-isValid

Reply:
We should ideally validate them in the datasource index and discard if invalid. BTW I was wondering if `DateTime.fromISO()` is too restricted, e.g. compared to `new Date()`?

Reply:
no, i think it's great, as all other code depoends on this too. So we should test and fix it in datasource as you suggested.